parkings/models/parking.py

```diff
@@ -79,18 +73,16 @@ class Parking(TimestampedModelMixin, UUIDPrimaryKeyMixin):
         verbose_name=_("parking end time"), db_index=True, null=True, blank=True,
     )
     domain = models.ForeignKey(
-        EnforcementDomain, on_delete=models.PROTECT,
-        related_name='parkings', null=True,)
+        EnforcementDomain, on_delete=models.PROTECT, null=True,)
     zone = models.ForeignKey(
-        PaymentZone, related_name='parkings', on_delete=models.PROTECT,
+        PaymentZone, on_delete=models.PROTECT,
         verbose_name=_("PaymentZone"), null=True, blank=True)
     is_disc_parking = models.BooleanField(verbose_name=_("disc parking"), default=False)
 
     objects = ParkingQuerySet.as_manager()
 
     class Meta:
-        verbose_name = _("parking")
-        verbose_name_plural = _("parkings")
+        abstract = True
```

Comment:
Maybe the methods after this could be on the Parking model rather than in the AbstractParking? Just thinking that ArchivedParking should probably not have the customization of the save method and they could have a different `__str__`.  Or the `__str__` could be in the base class and then just call super in the `ArchivedParking.__str__` and add string `archived` to it, or similar.